api/features_api.py

```diff
@@ -23,84 +23,120 @@
 from framework import rediscache
 from framework import users
 from internals.core_enums import *
-from internals.core_models import FeatureEntry, Stage
+from internals.core_models import FeatureEntry, MilestoneSet, Stage
 from internals.review_models import Gate
 from internals.data_types import VerboseFeatureDict
 from internals import feature_helpers
 from internals import search
 import settings
 
+
+# Data type for lists defining field data type information.
+FIELD_INFO_DATA_TYPE = list[tuple[str, str]]
+
+
 class FeaturesAPI(basehandlers.APIHandler):
   """Features are the the main records that we track."""
 
+  ############################
+  ###### Mutable Fields ######
```

Comment:
I love ASCII art, but if these kind of banners are seem needed, then maybe we should be doing something else to organize and clarify the code instead.   E.g., maybe this big constant should go into a server-side python version of form-field-specs.js.  That might also help with #3016.

Reply:
I agree that it would make sense to move the constants over to a specs file, and made the change accordingly. My hope was to separate the relevant methods used for each request with the big comments, but it's overall unnecessary, so I removed them. 🙂